src/components/ArticleComponents/AuthorList/Author/__tests__/index.test.tsx

```diff
@@ -6,13 +6,15 @@ describe('Author component', () => {
   it('renders correctly', () => {
     const username = 'test-author';
     const { container } = render(
-      <Author index={1} username={username} size="60" />
+      <Author index={1} username={username} githubImgLink="" />
```

Comment:
Need help with updating `githubImgLink` prop in test

Reply:
WDYM? What's the issue?

Reply:
Like I was not sure how to pass the `githubImgLink` value in the test case, snapshot got updated without `src` attribute for img tags. I was not sure whether we need to pass an empty string or the fallback img link.